drivers/include/periph/hwrng.h

```diff
@@ -59,6 +105,17 @@ void hwrng_init(void);
  */
 void hwrng_read(void *buf, unsigned int num);
 
+/**
+ * @brief   Read a single random word.
+ *
+ * The read function does not power on the HWRNG MCU peripheral, the caller
+ * is responsible for calling hwrng_poweron() / hwrng_poweroff() manually if
+ * HWRNG_HAS_POWERONOFF = 1.
+ *
+ * @return  The random number.
+ */
+uint32_t hwrand(void);
```

Comment:
Shouldn't this have a name more in line with the name space. To take from [`random`](https://doc.riot-os.org/group__sys__random.html), how about

```C
uint32_t hwrng_uint32(void);
```

Reply:
Yea probably. I just used that name since I had seen it used before in RIOT, but it looks like only esp32/esp8266 use it.

I'll change it then.

Reply:
Can I just use sed and foce-push or should I make a fixup commit for each individual commit?